src/components/card/LabelCard.jsx

```diff
@@ -0,0 +1,77 @@
+import React,{useState} from 'react'
+import { useNotes } from '../../context/NotesContext';
+import './LabelCard.css';
+
+function LabelCard() {
+    const [ label , setLabel ] = useState('');
+    const { note ,setNote ,labelArray , setLabelArray  } =useNotes();
+
+
+
+    const labelHandler =(e)=>{
+        const value = e.target.value;
+        setLabel(value);
+    }
+    console.log(label);
+    const addlabel =()=>{
+                console.log("click");
```

Comment:
You can remove console.log if not needed

Reply:
yes